blockchain-importer/src/Pos/BlockchainImporter/Tables/TxDetailsTable.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE Arrows                #-}
+{-# LANGUAGE FlexibleContexts      #-}
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE TemplateHaskell       #-}
+
+-- TODO: Add documentation.
```

Comment:
Should we add documentation in this in this PR?

Reply:
Not sure about the urgency of the merge. I'll ask Alan about this.

Reply:
Will add only basic documentation.